pkg/apis/apis/v1alpha1/types.go

```diff
@@ -142,6 +150,31 @@ type APIBindingStatus struct {
 	Conditions conditionsv1alpha1.Conditions `json:"conditions,omitempty"`
 }
 
+// These are valid conditions of APIBinding.
+const (
+	// APIExportValid is a condition for APIBinding that reflects that validity of the referenced APIExport.
+	APIExportValid conditionsv1alpha1.ConditionType = "APIExportValid"
+
+	// APIExportNotFoundReason TODO
+	APIExportNotFoundReason = "APIExportNotFound"
+	// GetErrorReason TODO
+	GetErrorReason = "GetError"
+
+	// CRDReady TODO
+	CRDReady conditionsv1alpha1.ConditionType = "CRDReady"
```

Comment:
what does Ready mean here? Estsablished? Created? Do we even want to leak to users that CRDs are backing the bindings? I think we don't want to.

Reply:
you could call this "Established" for the APIBinding, when all CRDs are created and Established.

Reply:
I don't think I'll have time to get the condition type/reason changes in today. Ok for follow-up?

Reply:
As far as I see being Established is part of ready. So I think it is fine.